Week2/homework/maartjes_work.js

```diff
@@ -45,3 +45,14 @@ const tuesday = [
 const tasks = monday.concat(tuesday);
 
 // Add your code here
+const inHours = tasks.map(hour => hour['duration']/60);
+console.log(inHours);
+
+const moreThanTwo = inHours.filter(hour => hour > 2);
```

Comment:
The assignment states: _Filter out everything that took less than two hours (i.e., remove from the collection)_, which implies that every task that took two hours or more should stay in. But in your case you leave out tasks that took exactly two hours.

The name `moreThanTwo` does not really reflect the fact that this will be an array. For instance, a name such as `filteredDurations` (note the plural 's' at the end) would reflect this better.